### PR TITLE
Add GitHub workflow for updating cacert.pem certificate bundle

### PR DESCRIPTION
**Pull request by heitbaum**
The workflow is scheduled to run every day at 11.30am UTC and it can optionally be triggered manually.

Originally from: https://github.com/Slicer/Slicer/blob/main/.github/workflows/update-slicer-certificate-bundle.yml

Adapted for LibreELEC/actions

Secret: `LIBREELECBOT_GITHUB_TOKEN`: is required with `public_repo` permissions in the actions repo to checkout and create the PR in LibreELEC/LibreELEC.tv

Reading materials:
- https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#push-pull-request-branches-to-a-fork

### CODE REVIEW

File: .github/workflows/update-cacert-pem-certificate-bundle.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            [1]: https://github.com/curl/curl/blob/master/scripts/mk-ca-bundle.pl
+            [2]: ${{ steps.latest_certdata.outputs.download_url }}
+            [3]: https://github.com/${{ github.repository }}/blob/${{ steps.head_branch.outputs.sha }}/.github/workflows/update-cacert-pem-certificate-bundle.yml
+          committer: LibreELEC Bot <libreelec-bot@libreelec.tv>
+          author: LibreELEC Bot <libreelec-bot@libreelec.tv>
+          reviewers: LibreELEC Bot <libreelec-bot@libreelec.tv>
+          labels: 'github-action'
+          signoff: false
+          #branch: bot/update-cacert-pem
+          #push-to-fork
```
Comment (CvH) [inline .github/workflows/update-cacert-pem-certificate-bundle.yml:87]:
```suggestion
```

File: .github/workflows/update-cacert-pem-certificate-bundle.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+            cacert.pem: update to ${{ steps.latest_certdata.outputs.authordate }}
+
+            This commit updates cacert.pem certificate bundle with [mk-ca-bundle.pl][1] script using the
+            content of [certdata][2] associated with ${{ env.CERTDATA_OWNER }}/${{ env.CERTDATA_REPO }}@${{ steps.latest_certdata.outputs.sha }},
+
+            It was auto-generated by the [update-cacert-pem-certificate-bundle][3] GitHub action workflow.
+
+            [1]: https://github.com/curl/curl/blob/master/scripts/mk-ca-bundle.pl
+            [2]: ${{ steps.latest_certdata.outputs.download_url }}
+            [3]: https://github.com/${{ github.repository }}/blob/${{ steps.head_branch.outputs.sha }}/.github/workflows/update-cacert-pem-certificate-bundle.yml
```
Comment (CvH) [inline .github/workflows/update-cacert-pem-certificate-bundle.yml:80]:
```suggestion
            cacert.pem: update to ${{ steps.latest_certdata.outputs.authordate }}

            This commit updates cacert.pem certificate bundle with [mk-ca-bundle.pl] script using the
            content of [certdata][1] associated with ${{ env.CERTDATA_OWNER }}/${{ env.CERTDATA_REPO }}@${{ steps.latest_certdata.outputs.sha }},

            [1]: ${{ steps.latest_certdata.outputs.download_url }}
}}/.github/workflows/update-cacert-pem-certificate-bundle.yml
```

Comment (CvH) [reply .github/workflows/update-cacert-pem-certificate-bundle.yml:80]: I think less chatty is good enough?

File: .github/workflows/update-cacert-pem-certificate-bundle.yml [hunk 3]
```diff
@@ -0,0 +1,97 @@
+# Originally from:
+# https://github.com/Slicer/Slicer/blob/main/.github/workflows/update-slicer-certificate-bundle.yml
+# Author: Jean-Christophe Fillion-Robin <jchris.fillionr@kitware.com>
+# Date:   Tue Sep 14 18:32:49 2021 -0400 - Wed Mar 16 02:45:45 2022 -0400
```
Comment (CvH) [inline .github/workflows/update-cacert-pem-certificate-bundle.yml:4]:
```suggestion
# Author: Jean-Christophe Fillion-Robin
```

Comment (CvH) [reply .github/workflows/update-cacert-pem-certificate-bundle.yml:4]: I would not add his email here.

**Review by CvH — CHANGES_REQUESTED** [review]

File: .github/workflows/update-cacert-pem-certificate-bundle.yml (hunk 3 above)
Comment (heitbaum) [reply .github/workflows/update-cacert-pem-certificate-bundle.yml]: all updated :-)

**Review by CvH — APPROVED** [review]

**Pull request merged**